Rollbar/RollbarLogger.cs

```diff
@@ -209,25 +219,55 @@ public ILogger Debug(object obj, IDictionary<string, object> custom = null)
 
         #endregion ILogger
 
-        private void Report(System.Exception e, ErrorLevel? level = ErrorLevel.Error, IDictionary<string, object> custom = null)
+        internal void Report(
+            System.Exception e, 
+            ErrorLevel? level = ErrorLevel.Error, 
+            IDictionary<string, object> custom = null, 
+            TimeSpan? timeout = null,
+            SemaphoreSlim signal = null
+            )
         {
             SendBodyAsync(new Body(e), level, custom);
```

Comment:
Shouldn't this pass the `timeout` and `signal` arguments?

Reply:
absolutely - should. thanks! fixed it.